native/src/main/java/io/ballerina/lib/soap/Soap.java

```diff
@@ -28,36 +28,33 @@ public class Soap {
 
     public static Object sendReceive11(Environment env, BObject soap11, Object body, BString action,
                                        BMap<BString, BString[]> headers, BString path, BTypedesc typeDesc) {
-        env.yieldAndRun(() -> {
+        return env.yieldAndRun(() -> {
             try {
                 Object[] arguments = new Object[]{body, action, headers, path};
-                Object result = env.getRuntime().call(soap11, REMOTE_FUNCTION, arguments);
+                Object result = env.getRuntime().callMethod(soap11, REMOTE_FUNCTION, null, arguments);
                 if (result instanceof BError) {
                     ((BError) result).printStackTrace();
-                    System.exit(1);
```

Comment:
what happen if we keep system.exit here

Reply:
The process exits and the testing is stopped as a result midway. Leading to a "there are test failures" error. When removed, the testing process successfully completes.

Reply:
Can u create an issue for this